wagtailio/packages/templates/packages/index.html

```diff
@@ -0,0 +1,23 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailadmin_tags i18n %}
+
+{% block titletag %}{% trans 'Styleguide' %}{% endblock %}
```

Comment:
`Styleguide` as the titletag – is this correct?

Reply:
👍 No, copy paste err. Fixed!